src/appStyles.ts

```diff
@@ -0,0 +1,7 @@
+import { css } from 'emotion';
+
+export const styles =
+	css`
+		display: flex;
+		flex-direction: row;
+	`;
```

Comment:
I suggest next code style:

```suggestion
export const styles = css`
  display: flex;
  flex-direction: row;
`; 
```